packages/studioCMS/src/integrations/studioCMSDashboard/routes/dashboard/pages/edit/pages/[...slug].astro

```diff
@@ -1,73 +1,83 @@
 ---
 // @ts-expect-error - Some types can only be imported from the Astro runtime
-import { Pages, SiteConfig, db, eq } from 'astro:db';
+import { PageData, PageContent, SiteConfig, db, eq } from 'astro:db';
 import { Layout } from 'studiocms-dashboard:components';
 import Config from 'virtual:studiocms/config';
-import { authHelper, urlGenFactory, type Locals } from 'studiocms:helpers';
-
-if (Config.includedIntegrations.useInoxSitemap) {
-    import('sitemap-ext:config').then((sitemap) => {
-        sitemap.default(false)
-    })
-}
-
-const { dashboardConfig: { developerConfig, dashboardRouteOverride } } = Config;
+import { authHelper, pluginList, type Locals, type PageDataAndContent } from 'studiocms:helpers';
+import { StudioCMSRoutes, getEditRoute } from 'studiocms-dashboard:routeMap';
+import type { SiteConfigResponse, pageDataReponse } from 'studiocms:components';
+
+const { dashboardConfig: { developerConfig } } = Config;
+
+const studioCMSPluginList = pluginList;
+
+const { 
+    authLinks: { 
+        loginURL 
+    }, 
+    mainLinks: { 
+        userProfile,
+        dashboardIndex,
+        pageEdit,
+        livePreviewBox
+    } 
+} = StudioCMSRoutes;
 
 const { slug } = Astro.params;
 
-const ConfigArray = await db.select().from(SiteConfig);
-const pageArrayed = await db.select().from(Pages).where(eq(Pages.slug, slug));
-
-const page = pageArrayed[0];
+const contextConfig: SiteConfigResponse = await db.select().from(SiteConfig).where(eq(SiteConfig.id, 1)).get();
```

Comment:
`1` too

Reply:
`1`, is the arbitrary id number that i assigned to the first entry to the SiteConfig Table in astroDB....  its also SUPPOSED to be the only entry in said table. and is created by the Database Setup page

Reply:
Done as of [3810634](https://github.com/astrolicious/studiocms/pull/77/commits/3810634c3e7fd7dd60ac90ef1dda6b1115e7e026)